Pinta.Core/Classes/DocumentSelection.cs

```diff
@@ -135,18 +135,14 @@ public DocumentSelection Clone ()
 	/// <returns>A Clipper Polygon collection.</returns>
 	public static List<List<IntPoint>> ConvertToPolygons (IReadOnlyList<IReadOnlyList<PointI>> pintaPolygonSet)
 	{
-		List<List<IntPoint>> newPolygons = new List<List<IntPoint>> ();
-
-		foreach (PointI[] pA in pintaPolygonSet) {
-			List<IntPoint> newPolygon = new List<IntPoint> ();
-
-			foreach (PointI p in pA) {
+		List<List<IntPoint>> newPolygons = new List<List<IntPoint>> (pintaPolygonSet.Count);
```

Comment:
Can this use the shorter `new (pintaPolygonSet.Count)`?

Reply:
Sure. I just pushed a commit